src/matrixrtc/CallMembership.ts

```diff
@@ -77,16 +91,27 @@ export class CallMembership {
     }
 
     public getAbsoluteExpiry(): number {
-        return this.createdTs() + this.data.expires;
+        if (this.data.expires) {
+            return this.createdTs() + this.data.expires;
+        } else {
+            // We know it exists because we checked for this in the constructor.
+            return this.data.expires_ts!;
+        }
     }
 
     // gets the expiry time of the event, converted into the device's local time
     public getLocalExpiry(): number {
-        const relativeCreationTime = this.parentEvent.getTs() - this.createdTs();
+        if (this.data.expires_ts) {
+            // With expires_ts we cannot convert to local time.
+            // TODO: Check the server timestamp and compute a diff to local time.
```

Comment:
do we already have a follow-up ticket for that TODO?

Reply:
As discussed I refactored this to use its own class for tracking the server/client diff with unimplmeneted clock sync diffs.
(It actally computes the diff once we get an event with an age property and from that point on its using this offset)

Reply:
The refactor is now on a different branch so that we can merge this "low risk"/"lightweight" version.

We will do the server-client-time sync properly here: https://github.com/matrix-org/matrix-js-sdk/pull/4038